docs/source/configuration/overview.mdx

```diff
@@ -109,77 +103,71 @@ without downtime
 </td>
 <td>
 
-Prints out the Apollo Router's version
+Prints out the Apollo Router's version.
 
 </td>
 </tr>
 
 </tbody>
 </table>
 
-## Configuration-awareness in your text editor
-
-The Apollo Router is able to produce a JSON schema for use in your text editor. It significantly eases the burden of making
-sure that your yaml file is correctly formatted, and even gives you content assist.
-
-To generate the schema use:
-```bash
-router --schema > configuration_schema.json
-```
-
-One you have generated the schema configure your text editor. Here are some instructions for well known editors:
-* [Emacs](https://emacs-lsp.github.io/lsp-mode/page/lsp-yaml)
-* [IntelliJ](https://www.jetbrains.com/help/idea/json.html#ws_json_using_schemas)
-* [Sublime](https://github.com/sublimelsp/LSP-yaml)
-* [Vim](https://github.com/Quramy/vison)
-* [VsCode](https://code.visualstudio.com/docs/languages/json#_json-schemas-and-settings)
-
 ## Configuration file
 
-Apollo Router takes an optional YAML configuration file as input via the `--config` option. All supported configuration options are shown below:
+The Apollo Router takes an optional YAML configuration file as input via the `--config` option.
 
 ### Listen address
 
-By default, the Apollo Router starts a HTTP server listening on `127.0.0.1:4000`. This can be
-changed in the configuration file:
+By default, the router starts an HTTP server that listens on `127.0.0.1:4000`. You can specify a different address like so:
 
 ```yaml title="router.yaml"
 #
 # server: Configuration of the HTTP server
 #
 server:
   # The socket address and port to listen on
-  # Defaults to 127.0.0.1:4000
-  listen: 127.0.0.1:4000
+  # (Defaults to 127.0.0.1:4000)
+  listen: 127.0.0.1:4001
 ```
 
 It can also listen on a Unix socket (not supported on Windows):
 
 ```yaml title="router.yaml"
 server:
-  # absolute path to a Unix socket
+  # Absolute path to a Unix socket
   listen: /tmp/router.sock
 
 ```
 
+### Overriding subgraph routing URLs
+
+By default, the Apollo Router extracts the routing URL for each of your subgraphs from the composed supergraph schema you provide it. In mose cases, no additional configuration is required.
+
+However, if you _do_ need to override a particular subgraph's routing URL (for example, to handle changing network topography), you can do so in your [YAML configuration file](./overview/#configuration-file) with the `override_subgraph_url` option:
+
+```yaml
+override_subgraph_url:
+  accounts: http://localhost:8080
+```
+
+In this example, the `accounts` subgraph URL is overridden to point to `http://localhost:8080`. The URL specified in the supergraph schema is ignored.
+
+Subgraphs _not_ included in the `override_subgraph_url` list continue to use the routing URL specified in the supergraph schema.
+
+### HTTP header rules
+
+See [Sending HTTP headers to subgraphs](./header-propagation/).
+
 ### Cross Origin Resource Sharing (CORS)
```

Comment:
```suggestion
### Cross-Origin Resource Sharing (CORS)
```

Just matching with the hyphenation that  typically shows browser output and MDN, etc.